CMakeLists.txt

```diff
@@ -26,11 +26,15 @@ endif()
 
 find_package(Threads REQUIRED)
 find_package(Python3 3.6 REQUIRED)
-if(NOT DISABLE_MPI)
-    find_package(MPI REQUIRED)
+if(DEFINED USE_MPI)
+    if(USE_MPI)
```

Comment:
I think a better name could be `WITH_MPI`

Reply:
Consider it done!